pkg/resources/testdata/TestAcc_GrantPrivilegesToShare/OnAllTablesInSchema/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "share_name" {
+  type = string
+}
```

Comment:
so the variables.tf doesn't actually need to be in its own file. maybe we should just put everything in test.tf. what do you think?

Reply:
Like in the comment https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2448#discussion_r1472122669 I would opt for not changing it on the spot in the single PR. We should aim for consistency, and currently in all other tests we have these two files separately.